app/assets/stylesheets/application.css

```diff
@@ -1,18 +1,89 @@
-/*
- * This is a manifest file that'll be compiled into application.css, which will include all the files
- * listed below.
- *
- * Any CSS and SCSS file within this directory, lib/assets/stylesheets, or any plugin's
- * vendor/assets/stylesheets directory can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
- *= require_tree .
- *= require_self
- */
- h1 {
+h1 {
   text-align: center;
 }
+/* @import url('https://fonts.googleapis.com/css?family=Numans'); */
+
+html,body{
+  background-image: url('http://getwallpapers.com/wallpaper/full/a/5/d/544750.jpg');
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 100%;
+  font-family: 'Numans', sans-serif;
+}
+
+.container{
+height: 100%;
+align-content: center;
+}
+
+.font{
+  font-size: 10px !important;
```

Comment:
Avoid using `!important` in css. It makes CSS complex and hard to change in future.
you can add another class to element to prioritize css.

Also pay attention to naming the classes - this class could've been `.font-10`